homeassistant/components/rest_command/__init__.py

```diff
@@ -97,20 +97,26 @@ def async_register_rest_command(name, command_config):
             request_url = template_url.async_render(variables=service.data)
             try:
                 with async_timeout.timeout(timeout):
-                    request = await getattr(websession, method)(
```

Comment:
The timeout handling is old. Can you update the style?
```
async with getattr()(...timeout=timeout) as response:
```

Reply:
@pvizeli I'm really, really new to Python, but I think I changed this correctly. Please take a look at the new version.